tests/scripts/run_profiler_regressions.sh

```diff
@@ -63,6 +63,7 @@ run_profiling_test(){
 
     TT_METAL_DEVICE_PROFILER=1 pytest $PROFILER_TEST_SCRIPTS_ROOT/test_device_profiler.py::test_custom_cycle_count -vvv
     TT_METAL_DEVICE_PROFILER=1 pytest $PROFILER_TEST_SCRIPTS_ROOT/test_device_profiler.py::test_full_buffer -vvv
+    TT_METAL_DEVICE_PROFILER=1 pytest $PROFILER_TEST_SCRIPTS_ROOT/test_device_profiler.py::test_dispatch_cores -vvv
```

Comment:
Hey per #7586 , could you refactor this pytest into single files that you can call so we get rid of `::` and `[]` in our shell scripts for pytest?